.travis.yml

```diff
@@ -9,6 +9,13 @@ notifications:
     on_start: change   # default: never
     on_cancel: always # default: always
     on_error: always # default: always
+cache:
+  directories:
+    - ~/.npm
+    - ~/.cache
```

Comment:
L12-15 seems out of scope, are they necessary? And if not what is the benefit and risk? We don't exactly need to make our build faster atm.

Reply:
Agreed. I've removed those lines.